[2023-07-02T08:23:13.419+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 1_init_once_seed_data.load_seed_data_once scheduled__2020-08-01T00:00:00+00:00 [queued]>
[2023-07-02T08:23:13.424+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 1_init_once_seed_data.load_seed_data_once scheduled__2020-08-01T00:00:00+00:00 [queued]>
[2023-07-02T08:23:13.425+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-02T08:23:13.434+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): load_seed_data_once> on 2020-08-01 00:00:00+00:00
[2023-07-02T08:23:13.437+0000] {standard_task_runner.py:57} INFO - Started process 86 to run task
[2023-07-02T08:23:13.441+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '1_init_once_seed_data', 'load_seed_data_once', 'scheduled__2020-08-01T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/init.py', '--cfg-path', '/tmp/tmpyml3m0nj']
[2023-07-02T08:23:13.442+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask load_seed_data_once
[2023-07-02T08:23:13.485+0000] {task_command.py:410} INFO - Running <TaskInstance: 1_init_once_seed_data.load_seed_data_once scheduled__2020-08-01T00:00:00+00:00 [running]> on host 7f1ae188f5a7
[2023-07-02T08:23:13.523+0000] {abstractoperator.py:598} ERROR - Exception rendering Jinja template for task 'load_seed_data_once', field 'env'. Template: {'dbt_user': '{{ var.value.dbt_user }}', 'dbt_password': '***', 'AIRFLOW__CELERY__RESULT_BACKEND': 'db+postgresql://***:***@postgres/***', 'DUMB_INIT_SETSID': '0', 'HOSTNAME': '7f1ae188f5a7', 'PYTHON_VERSION': '3.7.17', 'LANGUAGE': 'C.UTF-8', 'AIRFLOW_USER_HOME_DIR': '/home/***', 'ADDITIONAL_RUNTIME_APT_DEPS': '', 'PWD': '/opt/***', 'AIRFLOW__CELERY__BROKER_URL': 'redis://:@redis:6379/0', 'AIRFLOW_VERSION': '2.6.2', 'AIRFLOW__CORE__LOAD_EXAMPLES': 'false', 'AIRFLOW__API__AUTH_BACKENDS': '***.api.auth.backend.basic_auth,***.api.auth.backend.session', 'INSTALL_MSSQL_CLIENT': 'true', 'PYTHON_SETUPTOOLS_VERSION': '57.5.0', 'GUNICORN_CMD_ARGS': '--worker-tmp-dir /dev/shm', 'LD_PRELOAD': '/usr/lib/aarch64-linux-gnu/libstdc++.so.6', 'HOME': '/home/***', 'LANG': 'C.UTF-8', 'AIRFLOW_HOME': '/opt/***', 'GPG_KEY': '0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D', 'AIRFLOW__DATABASE__SQL_ALCHEMY_CONN': 'postgresql+psycopg2://***:***@postgres/***', 'AIRFLOW__CORE__EXECUTOR': 'CeleryExecutor', 'COMMIT_SHA': '', 'AIRFLOW_PIP_VERSION': '23.1.2', 'AIRFLOW__CORE__DAGS_ARE_PAUSED_AT_CREATION': 'true', 'ADDITIONAL_RUNTIME_APT_COMMAND': '', 'INSTALL_POSTGRES_CLIENT': 'true', 'SHLVL': '0', 'LC_MESSAGES': 'C.UTF-8', 'RUNTIME_APT_DEPS': '', 'PYTHON_PIP_VERSION': '23.0.1', 'RUNTIME_APT_COMMAND': 'echo', 'LD_LIBRARY_PATH': '/usr/local/lib', 'LC_CTYPE': 'C.UTF-8', 'PYTHON_GET_PIP_SHA256': '96461deced5c2a487ddc65207ec5a9cffeca0d34e7af7ea1afc470ff0d746207', '_AIRFLOW_WWW_USER_CREATE': 'true', 'AIRFLOW__CORE__SQL_ALCHEMY_CONN': 'postgresql+psycopg2://***:***@postgres/***', 'AIRFLOW_INSTALLATION_METHOD': '', 'LC_ALL': 'C.UTF-8', 'PYTHON_GET_PIP_URL': 'https://github.com/pypa/get-pip/raw/0d8570dc44796f4369b652222cf176b3db6ac70e/public/get-pip.py', 'INSTALL_MYSQL_CLIENT': 'true', 'PATH': '/root/bin:/home/***/.local/bin:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin', 'PYTHON_BASE_IMAGE': 'python:3.7-slim-bullseye', 'AIRFLOW_UID': '50000', '_AIRFLOW_WWW_USER_USERNAME': '***', 'BUILD_ID': '', 'AIRFLOW__CORE__FERNET_KEY': '', 'DEBIAN_FRONTEND': 'noninteractive', '_MP_FORK_LOGLEVEL_': '20', '_MP_FORK_LOGFILE_': '', '_MP_FORK_LOGFORMAT_': '[%(asctime)s: %(levelname)s/%(processName)s] %(message)s', 'CELERY_LOG_LEVEL': '20', 'CELERY_LOG_FILE': '', 'CELERY_LOG_REDIRECT': '1', 'CELERY_LOG_REDIRECT_LEVEL': 'WARNING', '_AIRFLOW_PARSING_CONTEXT_DAG_ID': '1_init_once_seed_data', '_AIRFLOW_PARSING_CONTEXT_TASK_ID': 'load_seed_data_once'}
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/abstractoperator.py", line 590, in _do_render_template_fields
    seen_oids,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/template/templater.py", line 168, in render_template
    return {k: self.render_template(v, context, jinja_env, oids) for k, v in value.items()}
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/template/templater.py", line 168, in <dictcomp>
    return {k: self.render_template(v, context, jinja_env, oids) for k, v in value.items()}
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/template/templater.py", line 156, in render_template
    return self._render(template, context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/abstractoperator.py", line 540, in _render
    return super()._render(template, context, dag=dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/template/templater.py", line 113, in _render
    return render_template_to_string(template, context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 288, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 283, in render_template
    return "".join(nodes)
  File "<template>", line 12, in root
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/context.py", line 103, in __getattr__
    self.var = Variable.get(key, deserialize_json=self._deserialize_json)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/variable.py", line 142, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable dbt_user does not exist'
[2023-07-02T08:23:13.526+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1407, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1531, in _execute_task_with_callbacks
    task_orig = self.render_templates(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 2179, in render_templates
    original_task.render_template_fields(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 1254, in render_template_fields
    self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 76, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/abstractoperator.py", line 590, in _do_render_template_fields
    seen_oids,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/template/templater.py", line 168, in render_template
    return {k: self.render_template(v, context, jinja_env, oids) for k, v in value.items()}
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/template/templater.py", line 168, in <dictcomp>
    return {k: self.render_template(v, context, jinja_env, oids) for k, v in value.items()}
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/template/templater.py", line 156, in render_template
    return self._render(template, context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/abstractoperator.py", line 540, in _render
    return super()._render(template, context, dag=dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/template/templater.py", line 113, in _render
    return render_template_to_string(template, context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 288, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 283, in render_template
    return "".join(nodes)
  File "<template>", line 12, in root
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/context.py", line 103, in __getattr__
    self.var = Variable.get(key, deserialize_json=self._deserialize_json)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/variable.py", line 142, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable dbt_user does not exist'
[2023-07-02T08:23:13.535+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=1_init_once_seed_data, task_id=load_seed_data_once, execution_date=20200801T000000, start_date=20230702T082313, end_date=20230702T082313
[2023-07-02T08:23:13.542+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 4 for task load_seed_data_once ('Variable dbt_user does not exist'; 86)
[2023-07-02T08:23:13.573+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-02T08:23:13.589+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
